app/models/taging.rb

```diff
@@ -0,0 +1,13 @@
+class Taging < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :task
+
+
+  def self.add_tag(user_id,task_id)
```

Comment:
@rakesh-1010 You should create tag using `task.tags.new` rather then using this class method.